deployments/porch/5-rbac.yaml

```diff
@@ -36,6 +36,9 @@ rules:
   - apiGroups: ["config.porch.kpt.dev"]
     resources: ["packagerevs", "packagerevs/status"]
     verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: ["apiregistration.k8s.io"]
+    resources: ["apiservices"]
+    verbs: ["get", "watch", "list"]
```

Comment:
Do we need all of these?

Reply:
no. you are right, we only need get. will fix.